src/languageservice/services/yamlHover.ts

```diff
@@ -92,6 +92,7 @@ export class YAMLHover {
 
     return this.schemaService.getSchemaForResource(document.uri, doc).then((schema) => {
       if (schema && node && !schema.errors.length) {
+        const descriptionsAreMarkdown = 'openapi' in schema.schema || 'swagger' in schema.schema;
```

Comment:
Move this to its own function so that it is easier for us to spot where to add new schemas in the future